README.md

```diff
@@ -7,45 +7,67 @@ Modules are polyphonic where applicable. Early releases and beta modules can be
 
 ## Building from code
 
-[VCV Rack source](https://github.com/VCVRack/Rack) is required. In the Rack build directory switch to 'plugins/' and
-
-```
-  git clone https://github.com/curlymorphic/StudioSixPlusOne.git
-  cd StudioSixPlusOne
-  make
-  ```
-  
-  New modules prior to release, may be in seperate branches, instruction will be given during testing.
+Instructions can be found in the VCV manual https://vcvrack.com/manual/Building#building-rack-plugins
   
  ## Modules
  
- ### KS Delay
+ ### Wallenda
  
- ![KS Delay screenshot](images/KSDelay.png)
+ ![Wallenda screenshot](images/Wallenda.png)
  
  A delay plugin designed for use when using [Karplus–Strong string synthesis](https://en.wikipedia.org/wiki/Karplus%E2%80%93Strong_string_synthesis) featuring:
  
  - Delay time tuned to pitch, via the V/oct input and the two tune knobs, Octave selector and fine tune in semitones
  - Feedback control, focusing on the area required for the string delay
- - Low Pass filter, to change the tone
- - Mix, to allow the blending of the noise impulse with the resulting string sound. I think of this like changing plectrum.
+ - Seven voice unison per polyphonic channel
+ - Stretch, the nature of strings gives a natural decay, this controls the length
  - Polyphonic, the number of channels is defined by the audio input.
  
- The main audio input should be triggered with noise, the color and duration of this has a large impact on the resulting sound. Try experimenting with short bursts, such as snare sounds, longer noise sounds with automated frequency can be used to emulate a bowed sound.
+ The main audio input should be triggered with noise, the color and duration of this has a large impact on the resulting sound. Try experimenting with short bursts, such as snare sounds, longer noise sounds with automated frequency can be used to emulate a bowed sound. All varations of sound input can be used to create sounds not envisaged by the author
  
 
 ### Maccomo
 
 ![Maccomo screenshot](images/Maccomo.png)
 
-An emulation of a Moog ladder filter, based on the descriptions and block diagrams in Will Pirkles book "Designing software synthesizers plugins in c++" featuring
+An emulation of a ladder filter, based on the descriptions and block diagrams in Will Pirkles book "Designing software synthesizers plugins in c++" featuring
```

Comment:
Title of the book should have proper casing.  Need punctuation at the end, maybe a colan.